conf/machine/qemuriscv32.conf

```diff
@@ -0,0 +1,15 @@
+#@TYPE: Machine
+#@NAME: qemuriscv32
+#@DESCRIPTION: QEMU RISC-V 32-bit machine
```

Comment:
This board is upstream: https://github.com/openembedded/openembedded-core/blob/master/meta/conf/machine/qemuriscv32.conf

Any changes should be sent there

Reply:
Thank you, @alistair23, for pointing that out! I’ll send any updates to the upstream repository, as suggested.